src/logic/safe/transactions/__tests__/gas.test.ts

```diff
@@ -12,19 +12,62 @@ describe('Get gas param', () => {
   afterAll(() => {
     setChainId(initialNetworkId)
   })
+  describe('isMaxFeeParams tests', () => {
+    it('should return maxFeePerGas for Mainnet', () => {
+      setChainId(CHAIN_ID.ETHEREUM)
+      expect(isMaxFeeParam()).toBe(true)
+    })
 
-  it('should return maxFeePerGas for Mainnet', () => {
-    setChainId(CHAIN_ID.ETHEREUM)
-    expect(getGasParam()).toBe('maxFeePerGas')
-  })
+    it('should return gasPrice for Arbitrum', () => {
+      setChainId(CHAIN_ID.ARBITRUM)
+      expect(isMaxFeeParam()).toBe(false)
+    })
 
-  it('should return gasPrice for Arbitrum', () => {
-    setChainId(CHAIN_ID.ARBITRUM)
-    expect(getGasParam()).toBe('gasPrice')
+    it('should return maxFeePerGas for Rinkeby', () => {
+      setChainId(CHAIN_ID.RINKEBY)
+      expect(isMaxFeeParam()).toBe(true)
+    })
   })
+  describe('createSendParams tests', () => {
```

Comment:
Do any of the values get converted to numbers here?

Reply:
Can you explain the question?

Reply:
I wasn't sure if you parsed any strings that, of course, should be tested but I just checked and [there aren't any](https://github.com/gnosis/safe-react/blob/cf92e1f86bc9783c26827789f54d59c643268fdb/src/logic/safe/transactions/gas.ts#L229). I'll resolve this.